tests/test_pipestat.py

```diff
@@ -2095,6 +2128,38 @@ def test_pipestat_iter(
 
 
 class TestMultiResultFiles:
+    # @pytest.mark.parametrize("backend", ["file"])
```

Comment:
If you won't use this code, and everything works fine, delete this code before merge

Reply:
Done